paparazzi-gradle-plugin/src/test/projects/compose/build.gradle

```diff
@@ -0,0 +1,31 @@
+plugins {
+  id 'com.android.library'
+  id 'kotlin-android'
+  id 'app.cash.paparazzi'
+}
+
+repositories {
+  maven {
+    url "file://${projectDir.absolutePath}/../../../../../build/localMaven"
+  }
+  mavenCentral()
+  //mavenLocal()
+  google()
+}
+
+android {
+  compileSdkVersion 30
+  defaultConfig {
+    minSdkVersion 25
+  }
+  buildFeatures {
+    compose true
+  }
+  composeOptions {
+    kotlinCompilerExtensionVersion '1.0.5'
```

Comment:
Can apps use other versions of Compose?

Reply:
i don't know, @JakeWharton any ideas?

Reply:
Yes. All newer compiler+runtime pairs are required to handle class files compiled by older, stable compiler+runtime pairs. Otherwise each new version of Compose would require _every_ library to recompile and ship new versions.